examples/emoji-color-cairo.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+#  pycairo/cairocffi-based emoji-color example - Copyright 2017 Hin-Tak Leung
+#  Distributed under the terms of the new BSD license.
+#
+#  This script demonstrates overlapping emojis.
+#
+
+import freetype
+import numpy as np
+from PIL import Image
+
+from cairo import ImageSurface, FORMAT_ARGB32, Context
+
+face = freetype.Face("Apple Color Emoji.ttf")
```

Comment:
Silly question, I can only find "Apple Color Emoji.ttc" on my system (Sierra 10.12.4), is there any difference ?